app-modules/interaction/src/Filament/Resources/InteractionDriverResource.php

```diff
@@ -66,6 +67,30 @@ public static function form(Form $form): Form
                     ->required()
                     ->maxLength(255)
                     ->placeholder('Interaction Driver Name'),
+                Toggle::make('is_default')
+                    ->label('Default')
+                    ->live()
+                    ->hint(function (?InteractionDriver $record, $state): ?string {
+                        if ($record?->is_default) {
+                            return null;
+                        }
+
+                        if (! $state) {
+                            return null;
+                        }
+
+                        $currentDefault = InteractionDriver::query()
+                            ->where('is_default', true)
+                            ->value('name');
+
+                        if (blank($currentDefault)) {
+                            return null;
+                        }
+
+                        return "The current default status is '{$currentDefault}', you are replacing it.";
+                    })
+                    ->hintColor('danger')
+                    ->columnStart(1),
```

Comment:
Because we cannot be 100% sure that this code will not run before the migrations to add the new column run you will need to add a Laravel Penant Feature flag here to only make this option visable/available once the Feature flag is `active`.

Once you have all your database migrations in place you will need to create a final `data_...` migration to activate the feature, which will switch the code to active and use this new column.

Take a look at some recent PRs by others on the team like the following for examples of how to do so. https://github.com/canyongbs/aidingapp/pull/75/files

You will need to do this everywhere your new code would interact with the new database changes.